okonomiyaki/file_formats/runtime.py

```diff
@@ -0,0 +1,201 @@
+import os.path
+import json
+
+import six
+import zipfile2
+
+from attr import attr, attributes
+from attr.validators import instance_of
+
+from okonomiyaki.errors import InvalidMetadata, UnsupportedMetadata
+from okonomiyaki.platforms import EPDPlatform, Platform
+from okonomiyaki.versions import MetadataVersion, SemanticVersion
+
+
+RuntimeVersion = SemanticVersion
+
+_METADATA_ARCNAME = "metadata/runtime.json"
+
+
+@attributes
+class RuntimeMetadataV1(object):
```

Comment:
Yes, otherewise how would you differentiate with say "1.1" metadata which would add some more attributes ?